invenio_app_ils/ui/src/semantic-ui/site/elements/container.overrides

```diff
@@ -2,6 +2,16 @@
          Site Overrides
 *******************************/
 
+/* Mobile */
+@media only screen and (max-width: @largestMobileScreen) {
+    .ui.container.fluid {
+        width: 100% !important;
+        margin-left: 0 !important;
+        margin-right: 0 !important;
```

Comment:
I was under the impression we would skip the `!important` overrides :/ with our custom theme

Reply:
Unfortunately I had to use it as semantic uses it and it is not possible to override it without `!important`